src/directives.ts

```diff
@@ -46,6 +34,41 @@ function specDirectiveTransformer(
     });
 }
 
+function versionDirectiveTransformer(
+    schema: GraphQLSchema,
+    directiveName: string,
+) {
+    const fieldMapper = objectTransformer(directiveName, schema);
+    return mapSchema(schema, {
+        [MapperKind.OBJECT_FIELD]: fieldMapper,
+        [MapperKind.ARGUMENT]: fieldMapper,
+        [MapperKind.ENUM_VALUE]: fieldMapper,
+        [MapperKind.ENUM_TYPE]: fieldMapper,
+        [MapperKind.FIELD]: fieldMapper,
+    });
+}
+
+const objectTransformer =
+    (directiveName: string, schema: GraphQLSchema) => (fieldConfig: any) => {
+        const apiDirective = getDirective(
+            schema,
+            fieldConfig,
+            directiveName,
+        )?.[0];
+        if (apiDirective) {
+            const versionText = `*since: ${apiDirective.since.join(' | ')}*`;
+            fieldConfig.version = apiDirective.since;
+            fieldConfig.description = fieldConfig.description
+                ? `${fieldConfig.description} <br/> <font color="red">${versionText}</font>`
+                : `${versionText}`;
```

Comment:
Do we not want to make versionText colored when there is no description, it should be consistent right ?

Also, I dont think red is the correct choice of color here. Should be light grey.

Reply:
There was a request to support `beta` tag too, should we include this here or it will be a separate iteration ?